src/pages/workspaces/workspace/tools/WorkflowView.js

```diff
@@ -205,33 +256,51 @@ const WorkflowView = _.flow(
           _.remove(s => s.includes(':'))
         )(_.keys(attributes))
       })
+      this.fetchInfo()
```

Comment:
There might be a timing issue here: The effects of `setState` don't always show up in the same tick, so the `savedConfig` might not be there when `fetchInfo` tries to grab it. We should probably either use the second 'callback' param to `setState`, or else explicitly pass the config as a param to `fetchInfo`